pgadmin-env.list

```diff
@@ -0,0 +1,3 @@
+PGADMIN_DEFAULT_EMAIL=jamie.maynard@me.com
```

Comment:
I think you checked this in by mistake

Reply:
Removed... Sorry about that... My Docker Postgres script needs tweaking.